GeneralizedMorphismsForCAP/gap/GeneralizedMorphismCategoryByCospans.gi

```diff
@@ -68,6 +68,13 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_GENERALIZED_MORPHISM_CATEGORY_BY_CO
         
         subobject2 := UniversalMorphismIntoDirectSum( pullback_diagram2 );
         
+        ## TODO: added more logic to make the following line obsolete
```

Comment:
It was unfortunately missing at the point when we added it here. Can this be added to the logic files?